extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -144,3 +144,179 @@ Pass the `origin key` to your front end. You might use this origin key to be abl
 
 > Note: First 2 steps are optional if origin key and payment methods has been already cached by the merchant server.
  
+## Step 3: Make a payment
+After the shopper submits their payment details or chooses to pay with a payment method that requires a redirection,
+the Adyen Web Components will generate a `makePaymentRequest`. Consult [Adyen documentation](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/payments) to see which parameters 
+are necessary for the current payment request.
+
+[Update CTP payment](https://docs.commercetools.com/http-api-projects-payments#update-payment) with `makePaymentRequest` custom field. 
+```json
```

Comment:
can you also add the example request as json before showing it in, check the step 2, it's not easy to read and understand a stringified JSON.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/335/commits/05cfc52eaa423642beb881fe874bcd33856cf155